src/OrchardCore.Themes/TheBlogTheme/Views/OrchardCore.Lucene/Search/Index.cshtml

```diff
@@ -0,0 +1,51 @@
+@model SearchIndexViewModel
```

Comment:
use a dymamic model instead and remove the reference. Or create an abstractions project with this view model.